job-service-db/src/main/resources/procedures/getTaskTableName.sql

```diff
@@ -20,15 +20,7 @@
  *  Description:
  *  Subtask structure is stored in dynamically-created tables.  This function builds the table name
  *  used for a given parent task.
- *
- *   - in_short_task_id: identification of the task - see
- *                       com.hpe.caf.services.job.util.JobTaskId#getShortId
  */
-CREATE OR REPLACE FUNCTION internal_get_task_table_name(
+DROP FUNCTION IF EXISTS internal_get_task_table_name(
     in_short_task_id VARCHAR(58)
-)
```

Comment:
Why not just update this function instead of introducing a new `internal_get_identity_based_task_table_name()` function?